test/PowerShellEditorServices.Test/Debugging/DebugServiceTests.cs

```diff
@@ -628,7 +628,7 @@ public async Task DebuggerSetsVariablesWithConversion()
             VariableDetailsBase[] variables = GetVariables(VariableContainerDetails.LocalScopeName);
 
             // Test set of a local string variable (not strongly typed but force conversion)
-            const string newStrValue = "False";
+            const string newStrValue = "\"False\"";
```

Comment:
This was erroneously changed by me previously, and is now back to [its original value](https://github.com/PowerShell/PowerShellEditorServices/blob/99b5b7e1a50f334bb0654e801484802c345a9cdf/test/PowerShellEditorServices.Test/Debugging/DebugServiceTests.cs#L760).